lib/identity_doc_auth/mock/doc_auth_mock_client.rb

```diff
@@ -71,16 +72,22 @@ def post_images(front_image:, back_image:, selfie_image:, liveness_checking_enab
         process_results(instance_id, liveness_checking_enabled, selfie_image)
       end
 
-      def get_results(instance_id:)
+      def get_results(instance_id:, liveness_enabled:)
         return mocked_response_for_method(__method__) if method_mocked?(__method__)
 
-        ResultResponseBuilder.new(self.class.last_uploaded_back_image).call
+        config = Config.new({
+          dpi_threshold: 290,
+          sharpness_threshold: 40,
+          glare_threshold: 40,
+        })
```

Comment:
Commonly I see us avoiding the braces, since the hash is the last (only) argument:

```suggestion
        config = Config.new(
          dpi_threshold: 290,
          sharpness_threshold: 40,
          glare_threshold: 40,
        )
```

Reply:
👍